### ISSUE TITLE
Error to load Assemblies dependency

### ISSUE THREAD

**Issue by lcrescenzo** [kahanu/System.Linq.Dynamic#60]
I have a project that depends on a third party dll and when DynamicLinq tries loaded the dll, an exception is generated not find the dll NUnit.

I do not have any functionality that depends on NUnit and do not intend to carry this dependence with my application.

Could please create a configuration for the DynamicLinq disregard or just carry specific dlls?

There is another way to solve the problem? without generating a new version of DynamicLinq?

The error occurs in DefaultDynamicLinqCustomTypeProvider class, method:
```
private static IEnumerable<Type> FindTypesMarkedWithAttribute()
{
			return from x in AppDomain.CurrentDomain.GetAssemblies().SelectMany((Assembly x) => x.GetTypes())
			where x.GetCustomAttributes(typeof(DynamicLinqTypeAttribute), false).Any<object>()
			select x;
		}
}
```

**Comment by StefH** [comment]
You can override this class, however I don't that functionality is define din this project.

See https://github.com/StefH/System.Linq.Dynamic.Core/blob/master/src/System.Linq.Dynamic.Core/CustomTypeProviders/DefaultDynamicLinqCustomTypeProvider.cs

**Comment by ryanvgates** [comment]
Can you provide the dll and a sample bit of code to reproduce the issue?

**Comment by lcrescenzo** [comment]
I implemented the IDynamicLinkCustomTypeProvider interface to load only the types that my application needs, as suggested by StefH.
 
I suggest you make an optional setting on app.config to load only the references specified to give greater flexibility to the component.

I tried to reproduce in a test application but could not succeed. The error occurs by trying to load a type that has a reference that is not being used by the application but that exists in the external reference.

Thank you.
